aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -322,43 +346,6 @@ window.addEventListener('pagehide', () => {
 });
 ```
 
-<!-- HOLD for reactivation at 5x
```

Comment:
What happened to this?

Reply:
I put it into a TXT file on my desktop. Safia and I will re-activate it somewhere for 6.0. I thought originally that it would be like a 5.1 thing, but that's not going to happen. It's a 6.0 scenario ...

https://github.com/dotnet/aspnetcore/issues/25705